libs/js-sdk/src/client.ts

```diff
@@ -34,23 +31,27 @@ export function createBrowserClient({
      */
     updateStrategy?: UpdateStrategies['kind'] | UpdateStrategies
 
-    store?: EffectiveFeatureStore
+    /**
+     * External state store is used to initialise the internal state store if retrieving the effective feature values from the API would fail.
+     * After initialisation the external state store will be updated but otherwise not used again.
+     *
+     */
+    externalStateStore?: ExternalStateStore<
+        EffectiveFeatureValue['value'] | undefined
+    >
     audiences: string[]
 
-    initialValues?: EffectiveFeatureValue[]
-
     environmentApiKey: string
 }): BrowserClient {
-    if (store && initialValues) {
-        throw new Error('Cannot specify both store and initialValues')
-    }
     const initialPromise = new PromiseCompletionSource<boolean>()
     const initialisedState: {
-        initialisedCallbacks: Array<(initialsed: boolean) => void>
+        initialisedCallbacks: Array<(initialised: boolean) => void>
         initialisedPromise: PromiseCompletionSource<boolean>
+        initialisedError: any
```

Comment:
I'd love for this to be `Error`, and we do the type narrowing to ensure it's an error type. But not needed